packages/cli/src/error_handler.test.ts

```diff
@@ -34,6 +35,14 @@ void describe('generateCommandFailureHandler', () => {
     mockExit.mock.resetCalls();
     mockEmitFailure.mock.resetCalls();
     mockEmitSuccess.mock.resetCalls();
+    resetHasError();
+  });
+
+  void it('does not print the same error multiple time', async () => {
+    const error = new Error('test error');
+    await generateCommandFailureHandler(parser, usageDataEmitter)('', error);
+    await generateCommandFailureHandler(parser, usageDataEmitter)('', error);
+    assert.equal(mockPrint.mock.callCount(), 1);
   });
```

Comment:
It doesn't look like the right fix for this problem. If the `generateCommandFailureHandler` is being called twice, I expect it to be executed twice.

What is the root cause of errors being printed twice? The code change here is not fixing the root cause rather hiding the problem (which can cause worse problems in the future)